cmd/cosign/cli/attach/sbom.go

```diff
@@ -60,6 +71,67 @@ func SBOMCmd(ctx context.Context, regOpts options.RegistryOptions, sbomRef strin
 	return remote.Write(dstRef, img, regOpts.GetRegistryClientOpts(ctx)...)
 }
 
+func SBOMCmdOCIExperimental(ctx context.Context, regOpts options.RegistryOptions, sbomRef string, sbomType ocitypes.MediaType, imageRef string) error {
```

Comment:
This doesn't seem to be used outside of this package, can we unexport it? (Same comment elsewhere)